graphql/src/domain/articles/entities/Article.ts

```diff
@@ -0,0 +1,18 @@
+export interface Tag {
+  name: string;
+}
```

Comment:
型定義は interface ではなく type を利用して

```suggestion
export type Tag = {
  name: string;
}
```

Reply:
ご指摘いただいた通り、interfaceをtypeに変更しました。

全ての型定義において：
- `interface` → `type` に統一
- Tag, User, Article など全ての型定義を修正
- TypeScriptの推奨パターンに準拠

https://github.com/ksugawara61/expo-template/commit/e791352

🤖 Generated by MCP